alphastats/loader/SpectronautLoader.py

```diff
@@ -147,28 +147,21 @@ def _read_spectronaut_file(
                     if bool(re.match(column_selection_regex, col))
                 ]
             ]
-            for column in df.columns:
-                try:
-                    if df[column].dtype == np.float64:
-                        continue
-                    df[column] = df[column].str.replace(",", ".").astype(float)
-                    print("converted", column, df[column].dtype)
-                except (ValueError, AttributeError):
-                    print("failed", column, df[column].dtype)
```

Comment:
I think this code was necessary to cover files, that use comma as decimal separator.